kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KcatClient.java

```diff
@@ -58,17 +56,7 @@ public void produceMessages(String topicName, String bootstrap, String message,
                 "--image=" + Constants.KCAT_CLIENT_IMAGE,
                 "--", "-b", bootstrap, "-l", "-t", topicName, "-P");
 
-        LOGGER.atInfo().setMessage("Executing command: {} for running kcat producer").addArgument(executableCommand).log();
-        ExecResult result = Exec.exec(String.valueOf(msg), executableCommand, Duration.ofSeconds(30), true, false, null);
-
-        String log = kubeClient().logsInSpecificNamespace(deployNamespace, name);
-        if (result.isSuccess()) {
-            LOGGER.atInfo().setMessage("kcat client produce log: {}").addArgument(log).log();
-        }
-        else {
-            LOGGER.atError().setMessage("error producing messages with kcat: {}").addArgument(log).log();
-            throw new KubeClusterException("error producing messages with kcat: " + log);
-        }
+        KafkaUtils.produceMessagesWithCmd(deployNamespace, executableCommand, String.valueOf(msg), name, KafkaClientType.KCAT.name().toLowerCase());
```

Comment:
Personally I'd pass the `KafkaClientType` into `produceMessagesWithCmd` and have it do `name().toLowerCase()` but I notice you don't do that with the strimzi client so maybe there is a good reason not to.